huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu

```diff
@@ -223,8 +223,6 @@ RUN HOME_DIR=/root && \
 
 # From Scallion open-source dependencies
 COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES /root/THIRD-PARTY-LICENSES
-# From generate_oss_compliance.sh
```

Comment:
Why are we modifying an image version which has already been released? Wouldn't it be safer to keep 1.0.3 as-is?

Reply:
ommited in new commit